superdesk/metadata/item.py

```diff
@@ -298,6 +298,11 @@
     'associations': {
         'type': 'dict',
     },
+    'alt_text': {
+        'type': 'string',
+        'nullable': True,
+        'mapping': not_analyzed,
```

Comment:
My comment was to change from "text" to "string" as the type of the mapping. I agree with Petr that "not_analysed" shouldn't be there either.